package.json

```diff
@@ -38,10 +41,14 @@
     "nodemon": "^2.0.6"
   },
   "dependencies": {
-    "@babel/preset-stage-2": "^7.8.3",
+    "chai-http": "^4.3.0",
     "cors": "^2.8.5",
+    "cross-env": "^7.0.2",
```

Comment:
why this package?

Reply:
it is not needed and i solved it

Reply:
@engmutabazi100  try to Read the use of this package you may need it